kafka/signed_token_redeem_handler.go

```diff
@@ -384,6 +381,21 @@ func SignedTokenRedeemHandler(
 	return nil
 }
 
+func issuerTimeIsNotValid(start *time.Time, end *time.Time) bool {
+	if start != nil && end != nil {
+		now := time.Now()
+
+		startIsNotZeroAndAfterNow := !start.IsZero() && start.After(now)
+		endIsNotZeroAndBeforeNow := !end.IsZero() && end.Before(now)
+
+		return startIsNotZeroAndAfterNow || endIsNotZeroAndBeforeNow
```

Comment:
These lines should be logically similar to:

https://github.com/brave-intl/challenge-bypass-server/blob/c952c2d5781055abd5e8b22f0a29b10f683b197b/kafka/signed_token_redeem_handler.go#L97-L102)